crates/nu-utils/src/default_files/scaffold_env.nu

```diff
@@ -1,5 +1,7 @@
 # env.nu
 #
+# version = "0.100.1"
```

Comment:
There's probably a buildscript somewhere that needs to be updated in order to do this.  Otherwise it's always going to be out-of-date.

When I get back to my desk I'll try to figure this out.

Reply:
But I'm not sure we should put a version in a file that simply has *comments* anyway :-)